arrow-libs/fx/arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/Fiber.kt

```diff
@@ -132,7 +139,14 @@ suspend fun <A> ForkScoped(
 }
 
 /** @see ForkScoped */
-// TODO provide proper deprecation annotation
+@Deprecated(
+  "Use Deferred with KotlinX Coroutines Structured Concurrency",
+  ReplaceWith(
+    "val scope = CoroutineScope(ctx); scope.async(ctx) { f() }.also { scope.launch { interruptWhen(); it.cancelAndJoin() } }",
```

Comment:
same here

```suggestion
    "async(ctx) { f() }.also { launch { interruptWhen(); it.cancelAndJoin() } }",
```

Reply:
idem, this has different semantics depending on the `CorotuineScope` used.